pints/_log_priors.py

```diff
@@ -499,7 +499,7 @@ class HalfCauchyLogPrior(pints.LogPrior):
     r"""
     Defines a 1-d half-Cauchy (log) prior with a given ``location`` and
     ``scale``. This is a Cauchy distribution that has been truncated to lie in
-    between :math:`[0,\infty]`, with pdf
+    between :math:`[0,\infty)`, with pdf
```

Comment:
Sure about this one? I don't quite see where the truncation happens in the code below?

Reply:
Hmm, this originally looked wrong to me just on the basis of notation, as in my experience, the square brackets at infinity ("[-∞,..." and "...,∞]") are only used for intervals on the extended reals, whereas the support of a cauchy distribution is always written (-∞, ∞).
I'd be happy to hear from @ben18785  who seems to have written the code here

Reply:
Looking at the code, (0, ∞) is the support of this distribution